<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelWindowClass</class>
 <widget class="QMainWindow" name="TravelWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1285</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1285</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TravelWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>944</width>
      <height>780</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="endCity">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>北京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上海</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>哈尔滨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>昆明</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>拉萨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>海口</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>郑州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>长沙</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>呼和浩特</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>成都</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>武汉</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="startCity">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>北京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上海</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>哈尔滨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>昆明</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>拉萨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>海口</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>郑州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>长沙</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>呼和浩特</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>成都</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>武汉</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="strategy">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>最少费用</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最少时间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>限时最少费用</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>策略选择:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>始发地:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>目的地:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="limitEdit">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>限时时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择途经城市:</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>190</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="startTime">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>330</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="displayFormat">
     <string>yyyy/MM/dd hh:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
    <property name="currentSectionIndex">
     <number>0</number>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="endTime">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>360</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="displayFormat">
     <string>yyyy/MM/dd hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>330</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>出发时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>360</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>到达时间:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>390</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成路线</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="answerEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>430</y>
      <width>311</width>
      <height>235</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stateButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>705</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询当前状态</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="stateEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>740</y>
      <width>311</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sumCost">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>676</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="sumTime">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>676</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>676</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>耗时:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>676</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>费用:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TravelWindow.qrc"/>
 </resources>
 <connections/>
</ui>
